### ISSUE TITLE
New pattern: mock globals

### ISSUE THREAD

**Issue by adriangodong** [adriangodong/jest-typescript-patterns#63]
Need to define pattern for mocking global objects like Panopto, SD, jQuery, etc., if we can't load the original library.

**Comment by adriangodong** [comment]
Use Object.assign:

```
    Object.defineProperty(global, "SD", { writable: false, value: { /* mock object here */} });
```

**Comment by adriangodong** [comment]
Fixed by 6d6ee89870a38e111601616aaf8040ee7cb8a70d
